src/usecases/counter/orchestrator.rs

```diff
@@ -0,0 +1,7 @@
+use crate::executor::BlsEigenlayerExecutor;
+use crate::orchestrator::generic::Orchestrator;
+use crate::usecases::counter::{CounterCreatorType, CounterHandler, CounterValidator};
+
+/// Type alias for counter orchestrator using the generic framework
+pub type CounterOrchestrator<C> =
+    Orchestrator<CounterCreatorType, BlsEigenlayerExecutor<CounterHandler>, CounterValidator, C>;
```

Comment:
is the orchestrator also a exectuor now?

Reply:
Nnot sure what you mean? It takes it as a parameter, but does that make it an executor? I don't think so.

<img width="362" height="196" alt="Screenshot 2025-08-25 at 10 35 48 AM" src="https://github.com/user-attachments/assets/5060c2ab-d65e-4ff4-a599-78a203ab8b60" />